test/searchModal/test.js

```diff
@@ -15,7 +15,10 @@
  *
  * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
  */
-define(['jquery', 'ui/searchModal', 'core/store', './mocks/searchStore'], function ($, searchModalFactory, store) {
+
+define(['jquery', 'ui/searchModal', 'core/store', './mocks/mocks'], function ($, searchModalFactory, store) {
```

Comment:
For your mocks you can use a JSON file

```suggestion
define(['jquery', 'ui/searchModal', 'core/store', 'json!test/ui/searchModal/mocks/mocks.json'], function ($, searchModalFactory, store, mocks) {
```